### PR TITLE
TA3 Schema Integration and Metadata Parsing

### PR DESCRIPTION
**Pull request by ajmuelle**
This isn't fully implemented (in particular the ingestion of NetCDF data) but we wanted to get the discussion going regarding how to merge in our new schemas for TA3 data along with the ingestion script.

### CODE REVIEW

**Comment by ajmuelle** [comment]
`populate_ta3_schema.py` (to be renamed) now lives at the [cdr repo](https://github.com/DARPA-CRITICALMAAS/cdr/blob/mtri/services/cdr/cdr/cli/populate_ta3_schema.py).

File: test_ta3_input.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from cdr_schemas.ta3_input import StackMetaData
```
Comment (jgawrilo) [inline test_ta3_input.py]: Should be changed now?

File: test_ta3_input.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from cdr_schemas.ta3_model import CMAModel
```
Comment (jgawrilo) [inline test_ta3_input.py]: Should be changed now?

**Comment by scotthaleen** [comment]
We have a docs generator script https://github.com/DARPA-CRITICALMAAS/cdr_schemas/blob/main/dev/docs.py#L59 

You can delete the ERD generator and update the docs script. 

https://github.com/DARPA-CRITICALMAAS/cdr_schemas?tab=readme-ov-file#development

If you run 
```
poetry install --with docs
poetry run docs
```
It should generate the ERD in README.  It can be a bit finicky

**Pull request merged**